python/doc/source/plasma.rst

```diff
@@ -0,0 +1,680 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+.. currentmodule:: pyarrow
+.. _io:
+
+The Plasma In-Memory Object Store
+=================================
+
+.. contents:: Contents
+  :depth: 3
+
+Installing Plasma
+-----------------
+
+Installation on Ubuntu
+^^^^^^^^^^^^^^^^^^^^^^
+
+The following install instructions have been tested for Ubuntu 16.04.
+
+
+First, install Anaconda in your terminal as follows. This will download 
+the Anaconda Linux installer and run it. Be sure to invoke the installer 
+with the ``bash`` command, whether or not you are using the Bash shell.
```

Comment:
Seems like we might merge these with the more general build instructions? Though I suspect that most users will obtain the plasma client via pip or conda packages